index-handlers/src/main/java/no/sikt/nva/nvi/index/model/document/NviCandidateIndexDocument.java

```diff
@@ -49,13 +51,56 @@ public static Builder builder() {
         return new Builder();
     }
 
+    @JsonIgnore
     public Approval getApprovalForInstitution(URI institutionId) {
         return approvals.stream()
                    .filter(approval -> approval.institutionId().equals(institutionId))
                    .findAny()
                    .orElseThrow();
     }
 
+    @JsonIgnore
+    public BigDecimal getPointsForContributorAffiliation(URI topLevelCristinOrg,
+                                                         NviContributor nviContributor,
+                                                         NviOrganization affiliation) {
+        return getApprovalForInstitution(topLevelCristinOrg).getPointsForAffiliation(nviContributor, affiliation);
+    }
+
+    @JsonIgnore
+    public String getReportingPeriodYear() {
```

Comment:
Are these getters necessary, or just for convenience? Could just do `reportingPeriod.year()` directly without a getter?

Reply:
My intention was to hide inner structure, but this is not necessary after refactoring and moving report representation creation to `NviCandidateIndexDocument`